blazor/pdfviewer-2/server-deployment/deploy-maui-windows.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: post
+title: Deploy SfPdfViewer in Blazor MAUI in windows | Syncfusion
+description: Checkout and learn here all about how to deploy SfPdfViewer in Blazor MAUI in windows in Syncfusion Blazor SfPdfViewer component and much more details.
+platform: Blazor
+control: SfPdfViewer
+documentation: ug
+---
+
+# How To Deploy SfPdfViewer in Blazor MAUI in windows
+
+This section briefly explains about how to deploy [Blazor SfPdfViewer](https://www.syncfusion.com/blazor-components/blazor-pdf-viewer) component in your Blazor MAUI in windows App using Visual Studio.
+
+## Prerequisites
+
+To use the MAUI project templates, install the Mobile development with .NET extension for Visual Studio. For more details, refer [here](https://docs.microsoft.com/en-us/dotnet/MAUI/get-started/installation).

```

Comment:
To use the MAUI project templates, install the Mobile development with the .NET extension for Visual Studio. For more details, refer to [here](https://docs.microsoft.com/en-us/dotnet/MAUI/get-started/installation).